contracts/AbstractConference.sol

```diff
@@ -324,4 +334,50 @@ contract AbstractConference is Conference, GroupAdmin {
         revert('tokenAddress must be impelmented in the child class');
     }
 
+    /* ERC721 implementation start */
+
+    function tokenURI(uint256 tokenId) public view returns (string memory) { 
+        return string(abi.encodePacked(baseTokenURI, Utils.addr2str(address(this)), '/', _tokenURI(tokenId)));
+    }
+    
+    /**
+     * @dev Internal returns an URI for a given token ID.
+     * Throws if the token ID does not exist. May return an empty string.
+     * @param tokenId uint256 ID of the token to query
+     */
+    function _tokenURI(uint256 tokenId) internal view returns (string memory) { 
+        require(_exists(tokenId), 'ERC721Metadata: URI query for nonexistent token');
+        return _tokenURIs[tokenId];
+    }
+
+    /**
+     * @dev Mints a new token.
+     * @param to The address that will own the minted token
+     * @param tokenId uint256 ID of the token to be minted
+     */
+    function mint(address to, uint256 tokenId) internal {
+        _mint(to, tokenId);
+        _tokenURIs[tokenId] = Utils.uint2str(tokenId);
+    }
+
+    function transferFrom(address from, address to, uint256 tokenId) public onlyActive {
+        require(!isRegistered(to), 'already registered');
+        super.transferFrom(from, to, tokenId);
+
+        Participant memory participant = participants[from];
+        participantsIndex[participant.index] = to;
+        participants[from] = Participant(0, address(0), participant.paid);
+        participants[to] = Participant(participant.index, address(uint160(to)), participant.paid);
+    }
+
+    function safeTransferFrom(address from, address to, uint256 tokenId) public {
+        safeTransferFrom(from, to, tokenId, '');
+    }
```

Comment:
Does this need to be overridden? Seems exactly the same as the parent class implementation

https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC721/ERC721.sol#L166
```
    function safeTransferFrom(
        address from,
        address to,
        uint256 tokenId
    ) public virtual override {
        safeTransferFrom(from, to, tokenId, "");
    }
```

Reply:
See the comment https://github.com/wearekickback/contracts/pull/76/files#r686677438